build/import/Packages.targets

```diff
@@ -124,10 +124,10 @@
     <PackageReference Update="Roslyn.Diagnostics.Analyzers"                                           Version="3.0.0-beta2.20059.3+77df2220" />
 
     <!-- NuGet -->
-    <PackageReference Update="NuGet.SolutionRestoreManager.Interop"                                   Version="5.6.0-preview.2.6532" />
-    <PackageReference Update="NuGet.VisualStudio"                                                     Version="5.6.0-preview.2.6532" />
-    <PackageReference Update="NuGet.ProjectModel"                                                     Version="5.6.0-preview.2.6532" />
-    <PackageReference Update="NuGet.Common"                                                           Version="5.6.0-preview.2.6532" />
+    <PackageReference Update="NuGet.SolutionRestoreManager.Interop"                                   Version="5.6.0-preview.2.6554" />
+    <PackageReference Update="NuGet.VisualStudio"                                                     Version="5.6.0-preview.2.6554" />
+    <PackageReference Update="NuGet.ProjectModel"                                                     Version="5.6.0-preview.2.6554" />
```

Comment:
We've avoid having other VS bound components depending on non-interop assemblies because of binding redirects. 
As you mentioned in a thread, I'd expect this to fail often given that there's no mechanism to keep this up to date at this point. 

The "instinctive" solution here is binding redirects, but that'd break many extensions and build tasks that might be depending on NuGet assemblies.

Reply:
Followed up on this via email.